.travis.yml

```diff
@@ -5,3 +5,5 @@ install:
     - true
 script:
     - mvn verify
+    - mvn clean verify -Pfunctional-tests
```

Comment:
instead of two runs:
~~~
    - mvn verify
    - mvn clean verify -Pfunctional-tests
~~~
just have one:
~~~
    - mvn clean verify -Pfunctional-tests
~~~